sql03rdOct/sql03rdOct.txt

```diff
@@ -0,0 +1,25 @@
+create database uber;
+use uber;
+create table riders(RiderID int primary key, Ridername varchar(30),RiderAge int,riderContact int,riderAddress varchar(60));
```

Comment:
format the queries.

use [](https://poorsql.com/)

Reply:
ok, updated.